gstreamer/src/toc.rs

```diff
@@ -214,7 +221,7 @@ impl TocEntryRef {
                 return None;
             }
 
-            Some(TagList::from_glib_borrow(tags as *const ffi::GstTagList))
+            Some(TagList::from_glib_none(tags as *const ffi::GstTagList))
```

Comment:
FWIW, this is exactly why the Ref types for the miniobjects exist and you can borrow them. But this seems not needed here and more complicated than anything else.